README.md

```diff
@@ -40,6 +40,38 @@ spark.read.format("dicomFile").option("includePixelData", true).load("/some/hdfs
 
 - `isDicom`: `true` if file was read as a DICOM file, `false` otherwise
 
+
+## De-identification
+
+The DICOM dataframe can be de-identified according to the Basic Confidentiality Profile in the [DICOM standard](https://dicom.nema.org/medical/dicom/current/output/html/part15.html#chapter_E). To use the de-identifier, do the following in scala:
+
+```scala
+import ai.kaiko.spark.dicom.deidentifier.DicomDeidentifier._
+
+var df = spark.read.format("dicomFile").load("/some/hdfs/path")
+df = deidentify(df)
```

Comment:
Such that it can used in ml pipelines one could write the `deidentify` function as a `org.apache.spark.ml.Transformer` with option as Parameters.

If eventually a training step will be required then rather `org.apache.spark.ml.Estimator`